install_config/topics/ocp_osp_sdn.adoc

```diff
@@ -0,0 +1,35 @@
+=== {product-title} SDN
+
+The default {product-title} SDN is
+xref:../architecture/networking/sdn.adoc#architecture-additional-concepts-sdn[OpenShiftSDN].
+There is another option: use xref:../install_config/configure_kuryrsdn.adoc#kuryr-sdn-and-openshift[Kuryr SDN].
+
+=== Kuryr SDN
+
+Kuryr is a CNI plugin using Neutron and Octavia to provide networking for pods
+and services. It is primarily designed for OpenShift clusters running on
+OpenStack virtual machines. Kuryr improves the network performance by plugin OCP
```

Comment:
OCP is not an approved shortname. You must remove all uses.
s/plugin OCP/plugging {product-title}